app/renderer/ui/pages/main/sections/smartContracts/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import {Route, Switch} from "react-router"
+
+import { TabMyContracts, TabEasyComposer, TabProEditor, TabMarketplace } from "./tabs"
+import TopBar from "app/renderer/ui/components/topBar/index"
+import { TopBarLinkProps } from "app/renderer/ui/components/commonTypes"
+import { SectionInfo } from "app/renderer/ui/pages/main/sections"
+
+const mainStyles = require("app/renderer/ui/pages/main/style.scss")
+const styles = require("./style.scss")
+
+/**
+ * @interface SmartContractsProps
+ */
+interface SmartContractsProps { }
+
+/**
+ * SmartContracts component
+ *
+ * @class SmartContracts
+ * @extends {React.Component<SmartContractsProps>}
+ */
+class SmartContracts extends React.Component<SmartContractsProps> {
+
+  // tslint:disable-next-line:prefer-function-over-method completed-docs
+  public render() {
+    const tabs = [TabMyContracts, TabEasyComposer, TabProEditor, TabMarketplace]
+    const topBarlinkProps: Array<TopBarLinkProps> = tabs.map(tab => (
```

Comment:
This could easily be taken out of the class and made available to all sections so that we can have a single `topBarLinkProps` factory parameterized over `tabs` and `sectionName`.